steps/kobiton-app-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,101 @@
+title: Kobiton App Upload
+summary: |
+  Uploads an application to Kobiton App Repo
+description: |
+  This step will take an .apk or .ipa file and upload it to the Kobiton App Repository. You can find your API key at [Kobiton](https://portal.kobiton.com/settings/keys)
+website: https://kobiton.com
+source_code_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo
+support_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo/issues
+published_at: 2021-12-02T11:32:43.775862+07:00
+source:
+  git: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo.git
+  commit: 0efcafb12186a942ac4cad573f6e5cbff71e4cfd
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
```

Comment:
What about flutter, ionic and other multi-platform projects?

Reply:
removed `project_type_tags ` to support all